satpy/etc/readers/avhrr_l1b_gaclac.yaml

```diff
@@ -25,6 +26,7 @@ datasets:
         calibration:
           reflectance:
             standard_name: reflectance
```

Comment:
I believe standard_names should be `toa_bidirectional_reflectance` and `toa_brightness_temperature`. @adybbroe could maybe verify that I'm not making things up. I don't remember what the radiance is supposed to be.

Reply:
Yes, good point.

Reply:
Although I checked a few other readers, we have `reflectance` and `brightness_temperature` there. So it might need another PR to fix all of these